pkg/controllers/externaloidc/generation/oauthapiserver/validation/validator_test.go

```
@@ -0,0 +1,247 @@
+	certPrivKey, err := rsa.GenerateKey(rand.Reader, 4096)
+	if err != nil {
+		return nil, err
+	}
+
+	certBytes, err := x509.CreateCertificate(rand.Reader, cert, caCert, &certPrivKey.PublicKey, caPrivateKey)
+	if err != nil {
+		return nil, err
+	}
+
+	certPEM := new(bytes.Buffer)
+	pem.Encode(certPEM, &pem.Block{
```

golangci-lint (2.12.2)

[error] Error return value of `pem.Encode` is not checked

(errcheck)